nipype/interfaces/base/traits_extension.py

```diff
@@ -465,3 +471,120 @@ class InputMultiObject(MultiObject):
 
 InputMultiPath = InputMultiObject
 OutputMultiPath = OutputMultiObject
+
+
+class Tuple(traits.BaseTuple):
+    """Defines a new type of Tuple trait that reports inner types."""
+
+    def init_fast_validator(self, *args):
+        """Set up the C-level fast validator."""
+        super(Tuple, self).init_fast_validator(*args)
+        self.fast_validate = args
+
+    def inner_traits(self):
```

Comment:
Below, `inner_traits` is accessed as an attribute. Should this be:
```suggestion
    @property
    def inner_traits(self):
```

Reply:
traits makes it look like a property without decorator: https://github.com/enthought/traits/blob/cf5f04b22eaf751f2bb8143a7b5945ea915b704a/traits/trait_types.py#L2601